.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```

Comment:
Typically, people will not execute this on their desktop.  I'd recommend changing this to "Environment" or "System Information" or something like that, and specify that this is the environment where it was run.  Note that the template should probably also be updated, as most tools will have the same problem.

Reply:
Agreed, I switched to "Environment". I'll also suggest that Jordan make this change in the template.